test/conformance/blue_green_test.go

```diff
@@ -125,7 +125,7 @@ func TestBlueGreenRoute(t *testing.T) {
 		clients.KubeClient,
 		logger,
 		greenDomain,
-		pkgTest.Retrying(pkgTest.MatchesAny, http.StatusNotFound),
+		pkgTest.Retrying(pkgTest.IsStatusOK(), http.StatusNotFound),
```

Comment:
nit: I know this is in `knative/pkg`, but could we make this not require the `()`?

Reply:
We can, would you be okay with me doing a cleanup run after this and #3221 have made it in?

Reply:
Yep